src/lib/y2storage/dialogs/inst_disk_proposal.rb

```diff
@@ -36,6 +36,20 @@ class InstDiskProposal < ::UI::InstallationDialog
       def initialize
         super
         textdomain "storage"
+        propose
+      end
+
+      def next_handler
+        if @devicegraph
```

Comment:
np: Maybe you should define an accessor (writer/reader or whatever). And don't forget to set the private.